docs/serving/creating-domain-mappings.md

```diff
@@ -5,23 +5,21 @@ weight: 64
 type: "docs"
```

Comment:
I just noticed this exist because I've searched the docs for such info and only found https://knative.dev/docs/serving/using-a-custom-domain-per-service/. Although, I think that the title fits both pages. Maybe they can be combined into one.